File: palindrome/palindrome.js
function isPalindrome(s) {
    for (let i = 0; i < s.length / 2; i++) {
      if (s[i] !== s[s.length - 1 - i]) {
        return false;
      }
    }
    return true;
  }
  
  const s = "katak";
  if (isPalindrome(s)) {
    console.log(s, "adalah palindrome");
  } else {
    console.log(s, "bukan palindrome");
  }
  

File: palindrome/palindrome.py
def is_palindrome(s):
  for i in range(len(s) // 2):
    if s[i] != s[-i-1]:
      return False
  return True

s = "katak"
if is_palindrome(s):
  print(s, "adalah palindrome")
else:
  print(s, "bukan palindrome")
